### PR TITLE
#289 feat: Use the same Lightbox gallery (with inter-linked images) i…

### PR DESCRIPTION
**Pull request by zeuros**
…n the profile's main page than on the gallery page

### CODE REVIEW

File: build/members/templates/profile.subcolumn_right.php [hunk 1]
```diff
@@ -286,21 +286,28 @@ class="float-right gray align-self-center"><i class="fa fa-flag" alt="<?=$words-
             <?php endif; ?>
         </h3>
 
-        <div class="p-2 d-flex flex-wrap justify-content-around">
+        <div class="p-2 d-flex flex-wrap justify-content-around"
+            style="overflow: scroll; max-height: 400px;">
```
Comment (thisismeonmounteverest) [inline build/members/templates/profile.subcolumn_right.php:290]: Isn't a max-height of 400px a bit small?

File: build/members/templates/profile.subcolumn_right.php [hunk 2]
```diff
@@ -286,21 +286,28 @@ class="float-right gray align-self-center"><i class="fa fa-flag" alt="<?=$words-
...
-        echo '<div><a href="gallery/show/image/'.$d->id.'">' .
-           '<img src="gallery/thumbimg?id='.$d->id.'"' .
-               ' alt="image"' .
-               ' style="height: 50px; width: 50px; margin: 1rem;"/>' .
-           '</a></div>';
-    }
-    ?>
+            foreach ($galleryImages as $galleryImage) {?>
+                <a href="gallery/img?id=<?= $galleryImage->id ?>" 
+                    style="max-width: 40%; display: flex; align-items: center; margin-bottom: 1rem;"
```
Comment (thisismeonmounteverest) [inline build/members/templates/profile.subcolumn_right.php:298]: Why limit the width to 40%? There's likely ample screen space and images are large.

Comment (zeuros) [reply build/members/templates/profile.subcolumn_right.php:298]: This is for x spacing between images / container borders

File: build/members/templates/profile.subcolumn_right.php (hunk 1 above)
Comment (zeuros) [reply build/members/templates/profile.subcolumn_right.php:290]: Hmmm, I don't know, which height would you suggest ?
Screenshot_20221025_235338

File: build/members/templates/profile.subcolumn_right.php (hunk 2 above)
Comment (thisismeonmounteverest) [reply build/members/templates/profile.subcolumn_right.php:298]: Obviously it's not a good idea to only look at code. 

If you want to ensure 2 columns using 50% with padding sounds like a good idea.

File: build/members/templates/profile.subcolumn_right.php (hunk 1 above)
Comment (thisismeonmounteverest) [reply build/members/templates/profile.subcolumn_right.php:290]: As stated below, I think this is fine as it is.

File: build/members/templates/profile.subcolumn_right.php (hunk 2 above)
Comment (thisismeonmounteverest) [reply build/members/templates/profile.subcolumn_right.php:298]: Let's see what others think about this update...

**Comment by thisismeonmounteverest** [comment]
@zeuros: It's now live on beta. 

Seeing it in use I observed the following: Thumbnails should all have the same height and the lightbox doesn't work as gallery for the up to 8 images but only shows the one clicked on.

Maybe do a second round?

**Pull request merged**